src/activities/Elsa.Activities.BlobStorage/Activities/DeleteBlob.cs

```diff
@@ -0,0 +1,42 @@
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using Elsa.ActivityResults;
+using Elsa.Attributes;
+using Elsa.Services;
+using Elsa.Services.Models;
+using Storage.Net.Blobs;
+
+// ReSharper disable once CheckNamespace
+namespace Elsa.Activities.BlobStorage
+{
+    [Action(
+        Category = "BlobStorage",
+        Outcomes = new[] { OutcomeNames.Done }
+    )]
+    public class DeleteBlob : Activity
+    {
+        public DeleteBlob(IBlobStorage storage)
+        {
+            _storage = storage;
+        }
+        private readonly IBlobStorage _storage;
+
+        [ActivityProperty(Hint = "The ID of the blob")]
+        public string BlobID { get; set; }
```

Comment:
From the designer's point of view, it would be better to remove the `BlobId` property and instead have just the `BlobIds` one. When configured with `UIHint = ActivityPropertyUIHints.MultiText`, `DefaultSyntax = SyntaxNames.Json` and `SupportedSyntaxes = new[] { SyntaxNames.Json, SyntaxNames.JavaScript, SyntaxNames.Liquid }`, the user by default will get a nice tagged input control.